src/openassetio-core-c/private/chandle.hpp

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2013-2022 The Foundry Visionmongers Ltd
+#pragma once
+
+#include <openassetio/export.h>  // For OPENASSETIO_VERSION
+
+namespace openassetio {
+inline namespace OPENASSETIO_VERSION {
+/**
+ * Provides utilities for converting between C++ types and C handles.
+ */
+namespace chandle {
```

Comment:
[question] Whats the convention here for the case style for word delimination, not having kebab, snake, or any camel variant seems like it might be hard to grok?

Reply:
Renamed to `handles` in d53e3389